documentation/howto/configure_remote_server.adoc

```diff
@@ -0,0 +1,62 @@
+
+The following instructions are used to set up a remote server for JBoss Enterprise Middleware application servers. A complete server definition requires a server adapter (or server) that allows the IDE to communicate with and manage the remote server.
+
+== Configuring a Remote Server
+
+. Click the `Servers` view. If the `Servers` view is not visible, click menu:Window[Show View > Server].
+. Use the appropriate instructions depending on the number of existing servers listed in the `Servers` tab:
+.. If there are no existing servers, click `No servers are available. Click this link to create a new server...`.
+.. If there are one or more existing servers, right-click an existing server and click menu:New[Server].
+. In the `New Server` wizard, add the relevant information for the following fields:
+.. From the `Select the server type` list, select a JBoss Enterprise Middleware application server.
+.. The `Server's host name` and `Server name` fields are completed by default. In the `Server name` field, you can type a custom name by which to identify the server in the `Servers` view.
+.. Click `Next >` to continue.
++
+.Define a New Remote Server
+image::remote_server_creation_screen.png[Define a New Remote Server]
++
+. Configure the required `Server Adapter` details:
+.. For the `The server is` field, click the `Remote` radio button option.
+.. For the `Controlled by` field, select either the `Filesystem and shell operations` or `Management Operations` radio button option depending on your requirements. 
++
+NOTE: If you select `Management Operations` for the `Controlled by` field, you must set up an admin user on the server by using the `$SERVER_HOME/bin/add-users.sh` script (for Linux, or the `$SERVER_HOME\bin\add-users.bat` file for Windows) and enter the same credentials in the server editor or during the server start.
++
+.. The `Server is externally managed. Assume server is started` field is used when the user wants to deploy the server but does not want the IDE to stop or start the server for them. Depending on the requirements, select this check box or leave it unchecked, as is default.
+.. A remote server can now be created without assigning a runtime to it. Depending on the requirements, select the Assign a runtime to this server check box (and select an existing runtime or create a new one) or leave the box unselected.
++
+.Create a New Server Adapter
+image::remote_create_new_server_adapter.png[Create a New Server Adapter]
++
+. Add the remote system integration details as follows:
+.. In the drop-down menu, select the appropriate host type in the `Host` field. The default host is `Local`. If required, use the `New Host` button to create a new host.
```

Comment:
Instead, could you explain the case when the user wants to connect to a remote host instead of 'Local' ? (or at least make it appear on the screenshot)